auth/basics/js.md

```diff
@@ -125,8 +129,9 @@ functions:
 
 ``` javascript
 import skygear from 'skygear';
-
-var otherData = { "interest": "reading" };
+let username = 'Ben';
+let password = '1234'; //very secure password
```

Comment:
@tensiuyan Very minor comment, but it would be great if you can help correct. Please add space after `//`, so it should be `// very secure password`.